### PR TITLE
feat: adds new tokens

### PR DESCRIPTION
**Pull request by Phani024**
Adds new tokens

Ticket: [BG-55007](https://bitgoinc.atlassian.net/browse/BG-55007)

### CODE REVIEW

**Review by barathcj — APPROVED** [review]

File: modules/statics/src/coins.ts [hunk 1]
```diff
@@ -298,6 +298,7 @@ export const coins = CoinMap.fromCoins([
   erc20('bidl', 'Blockbid Liquidity', 2, '0x5c7ec304a60ed545518085bb4aba156e8a7596f6', UnderlyingAsset.BIDL),
   erc20('bird', 'BirdCoin', 18, '0x026e62dded1a6ad07d93d39f96b9eabd59665e0d', UnderlyingAsset.BIRD),
   erc20('bit', 'BitDAO', 18, '0x1a4b46696b2bb4794eb3d4c26f1c55f9170fa4c5', UnderlyingAsset.BIT),
+  erc20('bnb', 'BNB Tokens', 18, '0xb8c77482e45f1f44de1745f52c74426c631bdd52', UnderlyingAsset.BNB),
```
Comment (mmcshinsky-bitgo) [inline modules/statics/src/coins.ts:301]:
```suggestion
  erc20('bnb', 'BNB Token', 18, '0xb8c77482e45f1f44de1745f52c74426c631bdd52', UnderlyingAsset.BNB),
```

File: modules/statics/src/coins.ts [hunk 2]
```diff
@@ -476,6 +477,7 @@ export const coins = CoinMap.fromCoins([
   erc20('gold', 'XBullion Token', 8, '0x0e573fdd3c5acf3a74f1775b0f9823a1a0e2b86b', UnderlyingAsset.GOLD),
   erc20('got', 'GOExchange', 18, '0xf11f2550769dac4226731b7732dd4e17e72b1b01', UnderlyingAsset.GOT),
   erc20('grt', 'The Graph', 18, '0xc944e90c64b2c07662a292be6244bdf05cda44a7', UnderlyingAsset.GRT),
+  erc20('gtc', 'Gitcoin', 18, '0xde30da39c46104798bb5aa3fe8b9e0e1f348163f', UnderlyingAsset.GTC),
```
Comment (mmcshinsky-bitgo) [inline modules/statics/src/coins.ts:480]: Are you using `Contract Overview` or `Token Tracker` for the names here? There looks to be inconsistencies between coin names in the PR.

File: modules/statics/src/coins.ts [hunk 3]
```diff
@@ -802,6 +804,7 @@ export const coins = CoinMap.fromCoins([
   erc20('wcfg', 'Wrapped Centrifuge', 18, '0xc221b7e65ffc80de234bbb6667abdd46593d34f0', UnderlyingAsset.WCFG),
   erc20('wec', 'Whole Earth Coin ', 18, '0xcc1a8bd438bebc4b2a885a34475bb974f2124317', UnderlyingAsset.WEC),
   erc20('wet', 'We Show Token', 18, '0x36d10c6800d569bb8c4fe284a05ffe3b752f972c', UnderlyingAsset.WET),
+  erc20('weth', 'Wrapped ETH', 18, '0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2', UnderlyingAsset.WETH),
```
Comment (mmcshinsky-bitgo) [inline modules/statics/src/coins.ts:807]:
```suggestion
  erc20('weth', 'Wrapped Ether', 18, '0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2', UnderlyingAsset.WETH),
```

File: modules/statics/src/coins.ts (hunk 2 above)
Comment (barathcj) [reply modules/statics/src/coins.ts:480]: I think we are just going by what is mentioned in the Jira ticket. @doodles92 can you confirm if the changes made are accurate

File: modules/statics/src/coins.ts (hunk 3 above)
Comment (barathcj) [reply modules/statics/src/coins.ts:807]: same comment as above

Comment (doodles92) [reply modules/statics/src/coins.ts:807]: This is correct; confirmed that "Wrapped Ether" is correct

File: modules/statics/src/coins.ts (hunk 2 above)
Comment (doodles92) [reply modules/statics/src/coins.ts:480]: This is correct; confirmed

File: modules/statics/src/coins.ts (hunk 1 above)
Comment (Phani024) [reply modules/statics/src/coins.ts:301]: @doodles92 Can you confirm this as well ^?

Comment (doodles92) [reply modules/statics/src/coins.ts:301]: Yes this is correct; should be "BNB Token"

**Comment by Phani024** [comment]
Hi @mmcshinsky-bitgo 
Could you please review this PR again and merge if it looks good?
Thanks

**Review by mmcshinsky-bitgo — APPROVED** [review]

**Pull request merged**